.azure-pipelines/templates/configure_azureml_agent.yml

```diff
@@ -4,6 +4,14 @@ steps:
   inputs:
     versionSpec: '3.9'
 
+- task: Cache@2
```

Comment:
Does caching only work for AzDo?  Is support for GitHub coming in another PR?

Reply:
as we agreed, two issues should be created. and we need to add links to this issues into this PR prior to merge. Later, we can pick GitHub issues in order to complete the sync between GitHub and DevOps

Reply:
Issue created: https://github.com/microsoft/dstoolkit-mlops-v2/issues/171